traffic_ops/install/bin/_postinstall

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/perl
-
+#!/usr/bin/env bash
```

Comment:
Did you change this Python code at all? I can't tell, the diff is so massive. I think if you use `git mv` to rename the files it'll make the diff cleaner, but I'm not sure since there was already a file with this name...

Reply:
I did use `git mv`, 

> Did you change this Python code at all?

Not in 83d69c3778, no. Running

```shell
git mv _postinstall _postinstall.pl;
git mv postinstall.py _postinstall;
```

yields

```shell
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        modified:   _postinstall
        new file:   _postinstall.pl
        deleted:    postinstall.py
```
, so I can see not wanting to do both steps in the same commit.

Rebased to split that commit into 2 `git mv` commits in ab87a7815e and 3179f52e43 (though the overall diff will still be just as big).

Reply:
The number of commits doesn't bother me. I just wish the diff was handled cleaner by git itself.